heat/core/tests/deviceselection.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import unittest
+import os
+import heat as ht
+
+envar = os.getenv("HEAT_USE_DEVICE", "cpu")
+
+if envar == "cpu":
+    ht.use_device("cpu")
+    torch_device = ht.cpu.torch_device
+    heat_device = None
+elif envar == "gpu" and torch.cuda.is_available():
+    ht.use_device("gpu")
+    torch.cuda.set_device(torch.device(ht.gpu.torch_device))
+    torch_device = ht.gpu.torch_device
+    heat_device = None
+elif envar == "lcpu" and torch.cuda.is_available():
+    ht.use_device("gpu")
+    torch.cuda.set_device(torch.device(ht.gpu.torch_device))
+    torch_device = ht.cpu.torch_device
+    heat_device = ht.cpu
+elif envar == "lgpu" and torch.cuda.is_available():
+    ht.use_device("cpu")
+    torch.cuda.set_device(torch.device(ht.gpu.torch_device))
+    torch_device = ht.gpu.torch_device
+    heat_device = ht.gpu
```

Comment:
I don't think that it should be a library function. It is only needed in the tests.

Reply:
i understand, but it goes against the programming model that we have everywhere else. and although this is only used in the unit tests at the moment, it may be useful as a tool for other developers if it is a library function.